### PR TITLE
scalafmt

### PR DESCRIPTION
**Pull request by philwills**
I ran this manually once and then did some fix-up of the `Param` definitions as, ironically, it made these less consistent.

I'm still not sure how I feel about this in general, but it does get rid of a decent chunk of noise and I think the enforcement of a newline in longer map/filter/groupBy type chains works well.

This doesn't integrate it into any build process, just does it as a one-off.

### CODE REVIEW

File: magenta-lib/src/main/scala/magenta/DeploymentPackage.scala [hunk 1]
```diff
@@ -6,28 +6,26 @@ import play.api.libs.json.JsValue
...
 }
 
-case class DeploymentPackage(
-  name: String,
-  pkgApps: Seq[App],
-  pkgSpecificData: Map[String, JsValue],
-  deploymentType: DeploymentType,
-  s3Package: S3Path,
-  legacyConfig: Boolean) {
+case class DeploymentPackage(name: String,
```
Comment (adamnfish) [inline magenta-lib/src/main/scala/magenta/DeploymentPackage.scala]: This syntax is an abomination

**Comment by adamnfish** [comment]
I realise this is super out of date but I stumbled across it.

Lots of little fixes, some meaningful improvements and a few things worsened. I'm broadly positive about the change in general, but I'm not convinced it's worth the impact on the history.

File: magenta-lib/src/main/scala/magenta/DeploymentPackage.scala (hunk 1 above)
Comment (philwills) [reply magenta-lib/src/main/scala/magenta/DeploymentPackage.scala]: Yea, I think that one might be fixable.

**Comment by philwills** [comment]
Looks like HTTP4S shared your concern, but decided it was worth it (https://github.com/http4s/http4s/commit/958753a3acdff6160ff65b3e74e62a1b585af568)

**Pull request closed without merging**